crates/rust/src/interface.rs

```diff
@@ -2931,21 +2798,3 @@ impl<'a, 'b> wit_bindgen_core::AnonymousTypeGenerator<'a> for AnonTypeGenerator<
         self.interface.push_str(">");
     }
 }
-
-fn stream_direct(ty: &Type) -> bool {
```

Comment:
I've replaced this with `is_list_canonical` which already existed but was somewhere else, so I refactored it to be available here.